cmd/check_container.go

```diff
@@ -191,11 +191,15 @@ var checkContainerCmd = &cobra.Command{
 				return err
 			}
 
-			// TODO: use the return values once we know what we need to display to the user
-			_, _, _, err = pyxisEngine.SubmitResults(certProject, certImage, rpmManifest, testResults)
+			_, certImage, _, err = pyxisEngine.SubmitResults(certProject, certImage, rpmManifest, testResults)
 			if err != nil {
 				return err
 			}
+
+			log.Infof("Test results have been submitted to Red Hat."+
```

Comment:
These might be better as individual log statements?

Reply:
I was thinking that as well, or was wondering if there was a way to formate like a `paragraph` for a log message? But I don't think that's a thing, even `\n` shows up as a literal.